src/gwt/src/org/rstudio/core/client/widget/DynamicIFrame.java

```diff
@@ -76,7 +76,8 @@ public void execute()
          {
             if (loadFrame.execute()) 
             {
-               Scheduler.get().scheduleFixedDelay(loadFrame, 50);
+               if (retryInterval_ < 500) retryInterval_ += 50;
```

Comment:
I think the `retryInterval_` should be reset whenever we set the URL (i.e. at the start of `pollForLoad`) so that each load gets its own backoff ramp.